draft-ietf-jsonpath-base.md

```diff
@@ -624,17 +626,18 @@ ALPHA           =  %x41-5A / %x61-7A    ; A-Z / a-z
 ~~~~
 
 Member names containing other characters than allowed by
```

Comment:
This isn't strictly in the PR, but since we are changing the sentence, better English would be "Member names containing characters other than allowed by".

Reply:
Fixed in next commit.